pylintrc_tweaks

```diff
@@ -2,3 +2,4 @@
 [MASTER]
 ignore = migrations
 load-plugins = edx_lint.pylint,pylint_django
+disable = model-missing-unicode
```

Comment:
Do you actually need this?  It looks like you fixed the one spot likely to trigger this warning.